### DISCUSSION TITLE
Adding JoltPhysics port to vcpkg

### DISCUSSION THREAD

**Discussion by JavierMatosD** [jrouwe/JoltPhysics#325] [Q&A] [answered]
Hi, 

I'm a developer on Microsoft's vcpkg team. If you haven't heard of vcpkg, it is a tool/ecosystem to helps manage c/c++ libraries on Windows, Linux, and MacOS. We have a user who would like to add your library to our catalog, but they would like to use your latest commit instead of your latest official release. They claim it is because your latest commit adds support for more platforms and improves your list files making it easier to integrate. However, it is the policy to respect upstream's release schedule. Hence, I have a couple of questions:

1. Do you approve of using the latest commit instead of the latest release?
2. Are you planning on making a release anytime soon, and if not, would you consider making an early release?

Best,
Javi

**Comment by jrouwe** [comment]
Hello,

I have heard of the vcpkg system and was planning to take a look at it.

I was working towards releasing a new version and I can probably release a new version this weekend.

Are there any patches needed for vcpkg that could be included in the main repository? (I assume it will be placed [here](https://github.com/microsoft/vcpkg/tree/master/ports)?). Can I take a look at the proposed config?

**Comment by jrouwe** [comment, marked as answer]
Found the PR: https://github.com/microsoft/vcpkg/pull/27715. Looks like there are no patches needed. So I will go ahead and make a release.

**Reply by JavierMatosD** [reply]
Thank you for the quick response! Please check out our repo if you are interested in adopting vcpkg, and feel free to reach out directly to me or anyone on the team with questions. :)
